OpenOversight/app/utils.py

```diff
@@ -29,40 +29,41 @@ def filter_by_form(form, officer_query):
     officer_query = officer_query.filter(db.or_(db.and_(Officer.birth_year <= min_birth_year,
                                                         Officer.birth_year >= max_birth_year),
                                                 Officer.birth_year == None))
+    # This handles the sorting upstream of pagination and pushes officers w/o tagged faces to the end of list
+    officer_query = officer_query.outerjoin(Face).order_by(Face.id)
```

Comment:
This is a bit on the bounds of my depth here, so if I'm wrong, feel free to correct me:
I don't know whether postgres enforces strict ordering of returned values, but an issue I've seen elsewhere in pagination code that called out to other RDBMSes is that the data can be returned in a way that causes duplicates and missing entries if the order_by clause is non-unique.
I know that `face` has a FK to `officer`, but i see nothing in the app right now that would make that mapping 1-to-1: many `officer`s (including ones without images but I don't know if any do right now) don't have a `face` entry.
So in a scenario where we have, say, 21 un-`face`d officers, since we're making the queries at different times there's nothing in the `order_by` clause to make that ordering guaranteed.
I see some model changes above to possibly address this, but I'm not convinced they're sufficient and it'd be an insidious and difficult-to-catch bug.